web/src/main/webapp/xml/csw/capabilities.xml

```diff
@@ -8,37 +8,37 @@
 
   <ows:ServiceIdentification>
     <ows:Title>$TITLE</ows:Title>
-    <ows:Abstract>$ABSTRACT</ows:Abstract>
+    <ows:Abstract></ows:Abstract>
```

Comment:
I see this file is used if no record is selected, what looks good. Maybe can be relevant in the `iso19139` samples to provide an example of a CSW service record and another for CSW INSPIRE service record, to facilitate the task for users.

Reply:
See https://github.com/metadata101/iso19115-3.2018/pull/29